rtshell/rtwatch.py

```diff
@@ -0,0 +1,228 @@
+#!/usr/bin/env python2
+# -*- Python -*-
+# -*- coding: utf-8 -*-
+
+'''rtshell
+
+Copyright (C) 2009-2015
+    Geoffrey Biggs
+    RT-Synthesis Research Group
+    Intelligent Systems Research Institute,
+    National Institute of Advanced Industrial Science and Technology (AIST),
+    Japan
+    All rights reserved.
+Licensed under the GNU Lesser General Public License version 3.
+http://www.gnu.org/licenses/lgpl-3.0.en.html
+
+Implementation of the command to watch component events.
+
+'''
+
+from __future__ import print_function
+
+import optparse
+import os
+import os.path
+import rtctree.exceptions
+import rtctree.tree
+import rtctree.path
+import RTC
+import sys
+import traceback
+import time
+import atexit
+
+from rtshell import path
+from rtshell import rts_exceptions
+import rtshell
+
+counter = 0
+
+def rtc_status_cb(event, args, args2):
+    global counter
+    sevent = args[1]
+    if sevent == rtctree.component.Component.INACTIVE:
```

Comment:
Use a dictionary to map the event type to strings rather than a big `if` statement.